### ISSUE TITLE
[Layout]: WCAG and other issues

### ISSUE THREAD

**Issue by ivarkallejarv** [TEDI-Design-System/react#110]
## Problem
- Use Layout component, add navigable elements via sideNav and navigate to said elements using JAWS. JAWS doesn't read them out as clickable elements
![Image](https://github.com/user-attachments/assets/7bb91244-60be-408f-9bb8-75f209f74df6)
